crates/ruff_linter/src/rules/flake8_annotations/rules/definition.rs

```diff
@@ -469,7 +469,7 @@ impl Violation for MissingReturnTypeClassMethod {
 /// ```
 ///
 /// ## References
-/// - [PEP 484](https://www.python.org/dev/peps/pep-0484/#the-any-type)
+/// - [PEP 484: The `Any` type](https://peps.python.org/pep-0484/#the-any-type)
```

Comment:
https://typing.readthedocs.io/en/latest/spec/special-types.html#any might be an even better place to link to than PEP 484, since the PEP is a historical document whereas the spec is living documentation that's kept updated

Reply:
Agreed, updated the link.